drop-down-common.ts

```diff
@@ -154,4 +156,16 @@ export const hintProperty = new Property<DropDownBase, string>({
     name: "hint",
     defaultValue: ""
 });
-hintProperty.register(DropDownBase);
\ No newline at end of file
+hintProperty.register(DropDownBase);
+
+export const itemsTextAlignmentProperty = new Property<DropDownBase, string>({
```

Comment:
this property should be changed. Its type should be `TextAlignment` not string. Also it should have a `valueConverter` and default value should be `initial`. Take a look at how it is done in the NS core modules:
https://github.com/NativeScript/NativeScript/blob/46705ee33278e8f9ec04b00ea592dd65ba855e74/tns-core-modules/ui/text-base/text-base-common.ts#L198-L200